packages/cli/src/config/settings.ts

```diff
@@ -70,6 +70,7 @@ export interface Settings {
   allowMCPServers?: string[];
   excludeMCPServers?: string[];
   showMemoryUsage?: boolean;
+  showToolUsage?: boolean;
```

Comment:
I think the name here could be misleading, since it's a setting to show tool usage for non interactive tool usage not all tool usages. For example someone could be confused and think it will disable showing tool usage in interactive mode.